spec/syntax.md

```diff
@@ -669,6 +651,45 @@ reserved-char = %x00-08        ; omit HTAB and LF
               / %xE000-10FFFF
 ```
 
+## Markup
+
+**_<dfn>Markup</dfn>_** _placeholders_ are _pattern_ parts
+that can be used to represent non-language parts of a _message_,
+such as inline elements or styling that should apply to a span of parts.
+Markup comes in three forms:
+
+**_<dfn>Markup-open</dfn>_** starts with `#` and
+is a _pattern_ part starting a span.
+It MAY include _options_, but these are not required.
+
+If a _markup-open_ includes a `/` immediately before its closing `}`,
+it is a self-closing **_<dfn>markup-standalone</dfn>_** part
+representing a standalone element within a message,
+such as an inline image or any other opaque content.
```

Comment:
This should be the same as the other two types.

```suggestion
**_<dfn>Markup-standalone</dfn>_** starts with U+0023 NUMBER SIGN `#`
and has a U+002F SOLIDUS `/` immediately before its closing `}`
representing a self-closing or standalone element within the _message_.
It MAY include _options_, but these are not required.
```